services/graphql-server/src/graphql/definitions/email/newsletter.js

```diff
@@ -150,4 +154,11 @@ input EmailNewsletterSortInput {
   order: SortOrder = desc
 }
 
+input UpdateEmailNewsletterProviderInput {
+  id: ObjectID!
+  type: String
+  providerId: Int
```

Comment:
This should be a `String`, matching the Base [YAML definition](https://github.com/parameter1/base-platform/blob/master/app/config/platform/models/modelspaces/email/deployment.yml#L91) and GraphQL [type definitions](https://github.com/parameter1/base-cms/blob/76dbc184cc9e4c817b8c8d53151f309fc26bf817/services/graphql-server/src/graphql/definitions/email/newsletter.js#L89)